win32/src/async_socket_win32.c

```diff
@@ -726,6 +739,10 @@ int async_socket_receive_async(ASYNC_SOCKET_HANDLE async_socket, ASYNC_SOCKET_BU
                             /* Codes_SRS_ASYNC_SOCKET_WIN32_01_081: [ An asynchronous IO shall be started by calling StartThreadpoolIo. ]*/
                             StartThreadpoolIo(async_socket->tp_io);
 
+#ifdef ENABLE_SOCKET_LOGGING
+                            LogVerbose("Starting receive at %lf", timer_global_get_elapsed_us());

```

Comment:
>LogVerbose("Starting receive at %lf", timer_global_get_elapsed_us()) [](http://example.com/codeflow?start=28&length=68)

Do you want to have a log after we receive to say we have received xx amount?

Reply:
We have that log in ln 158.

---
In reply to: [654743078](https://github.com/Azure/c-pal/pull/95#discussion_r654743078) [](ancestors = 654743078)